.github/workflows/test-e2e.yml

```
@@ -28,3 +28,18 @@ jobs:
           echo "${KUBECONFIG_FILE}" > $HOME/.kube/config
       - name: Run e2e tests
         run: make test-e2e
+
+      - name: Notify slack on failure
```

[optional] IMO a more explanatory title

```suggestion
      - name: If the e2e test fails, notify Slack channel #e2e-test-failures
```